smclib/include/smclib/statemap.hpp

```diff
@@ -52,7 +52,6 @@
 #endif  // SMC_NO_EXCEPTIONS
 #include <cstdio>
 #elif defined(WIN32)
-#include <windows.h>
```

Comment:
I know your point is only to fix the Windows problem, but I kind of feel like this nest of ifdefs can be replaced by:

```
#if defined(SMC_NO_EXCEPTIONS)
#include <cassert>
#include <cstdio>
#else
#include <cstring>
#include <stdexcept>
#endif
```

What do you think?

Reply:
Good point, I did not actually check the code context, I implemented the suggested solution in https://github.com/ros/bond_core/pull/109/commits/0a2720bd7749142e9b2d57b9545dea23aee7d35f .